### DISCUSSION TITLE
宝塔部署异常

### DISCUSSION THREAD

**Discussion by Cyrene963** [coaidev/coai#298] [Q&A | 问题解答] [answered]
[//]: # (方框内删除已有的空格，填 x 号)
+ [] 我已确认目前没有类似 issue
+ [] 我已确认我已升级到最新版本
+ [] 我已完整浏览项目 README 和项目文档并未找到解决方案
+ [] 我理解并愿意跟进此 issue，协助测试和提供反馈
+ [] 我将以礼貌和尊重的态度提问，不得使用不文明用语 (包括在此发布评论的所有人同样适用, 不遵守的人将被 block)
+ [] 我理解并认可上述内容，并理解项目维护者精力有限，**不遵循规则的 issue 可能会被无视或直接关闭**

**问题描述**
Deployment on one-click BTpanel is unsuccessful
**复现步骤**
Set my domain name and reverse proxy.After opening my website via domain name,it can't login in whth my root account when it keep warning can't find the port of redis.I've tried a lot of time to rejust the network of the docker-compose and restart the redis.Unfortunately,it's unsuccessful.Nevertheless,when I delete all the things and deploy it with docker-compose.It's just successful with one try.Hence,I propose there's some problem with deploying in BTpanel.Plz check this and atempt!
**预期结果**
Succeed in deploying with BTpanel
**日志信息**

**相关截图 (如果有)**

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
没试过，宝塔mysql/redis拉取下来了吗

[a comment by an automated account is not shown]

**Comment by Cyrene963** [comment]
> 没试过，宝塔mysql/redis拉取下来了吗

怎么讲起中文来了 宝塔那些数据库都安装了的 就是容器之间Chatnio和redis之间互连不了啊 望作者再亲测一下

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
有空我去问一下

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
确实会有docker dns无法解析的问题，等我看看是哪里的network没调好

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
![image](https://github.com/user-attachments/assets/4f1bf45a-9ce1-47f7-9721-fe5d1fd69ec1)

排查到问题了，他们写应用商店的时候redis的env漏了chatnio-前缀😅....... 明天找一下他们修一下就好了。

**Comment by zmh-program** [comment, marked as answer]
OK了，解决方法：
1. 卸载原先不可用的CoAI应用
2. 返回应用市场，点击右上角**更新应用列表**进行更新
3. 重新按照原步骤安装

![image](https://github.com/user-attachments/assets/7532ff57-2653-4228-83bf-9a189c54553c)

[a comment by an automated account is not shown]

**Comment by zmh-program** [comment]
@Brave963 @chengtx809

**Comment by zmh-program** [comment]
好了，已测试成功

[a comment by an automated account is not shown]
